zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/MappingCreateProcessor.java

```diff
@@ -26,8 +26,11 @@
 
 public class MappingCreateProcessor implements DistributedTypedRecordProcessor<MappingRecord> {
 
-  private static final String MAPPING_ALREADY_EXISTS_ERROR_MESSAGE =
+  private static final String MAPPING_SAME_CLAIM_ALREADY_EXISTS_ERROR_MESSAGE =
       "Expected to create mapping with claimName '%s' and claimValue '%s', but a mapping with this claim already exists.";
+  private static final String MAPPING_SAME_ID_ALREADY_EXISTS_ERROR_MESSAGE =
+      "Expected to create mapping with id '%s' but a mapping with this id already exists.";
```

Comment:
```suggestion
      "Expected to create mapping with id '%s', but a mapping with this id already exists.";
```

🙃 Let's add a comma here to keep it aligned with the other error message